doc/whats_new/v1.3.rst

```diff
@@ -397,7 +397,7 @@ Changelog
 
 - |Fix| :func:`metric.ndcg_score` now gives a meaningful error message for input of
   length 1.
-  :pr:`25672` by :user:`Lene Preuss <lene>` and :user:`Wei-Chun Chu <wcchu>`.
+  :pr:`25672` by :user:`Lene Preuss <lens>` and :user:`Wei-Chun Chu <wcchu>`.
```

Comment:
This one should not be changed this is a github user handle. Not sure what option there is to tell codespell to leave this one alone (and maybe similar ones like `<some-text-here-for-a-github-handle>` in particular in `whats_new` rst files).